tests/unit/test_get_clade_list.py

```diff
@@ -2,7 +2,11 @@
 from unittest.mock import MagicMock, patch
 
 import pytest
-from cladetime.get_clade_list import main
+
+pytest.importorskip(
```

Comment:
I really like this parameterized test for checking our "get_clade_list" logic. However, that script now (rightfully) lives in the variant nowcast hub.

Need to do some more thinking about how to incorporate these tests in the other repo, where we don't necessary have a python/pytest environment set up.

In the meantime, added this code to skip the test file.